tests/test_api.py

```diff
@@ -78,3 +78,43 @@ def test_release_api(app, test_user, github_api):
 
         assert valid_remote_file_contents is not None
         assert valid_remote_file_contents.decoded["name"] == "test.py"
+
+
+def test_test_zipball(app, test_user, github_api):
+    api = GitHubAPI(test_user.id)
+    api.init_account()
+    repo_id = 2
+    repo_name = "repo-2"
+
+    # Create a repo hook
+    hook_created = api.create_hook(repo_id=repo_id, repo_name=repo_name)
+    assert hook_created
+
+    headers = [("Content-Type", "application/json")]
+
+    payload = github_payload_fixture(
+        "auser", repo_name, repo_id, tag="v1.0-tag-and-branch"
+    )
+    with app.test_request_context(headers=headers, data=json.dumps(payload)):
+        event = Event.create(
+            receiver_id="github",
+            user_id=test_user.id,
+        )
+        release = Release(
+            release_id=payload["release"]["id"],
+            tag=event.payload["release"]["tag_name"],
+            repository_id=repo_id,
+            event=event,
+            status=ReleaseStatus.RECEIVED,
+        )
+        # Idea is to test the public interface of GithubRelease
+        gh = GitHubRelease(release)
```

Comment:
I think that some of the boilerplate above this line could be removed, but I'm not 100% of the implications.